indico/modules/events/editing/client/js/models.js

```diff
@@ -57,6 +57,7 @@ export const EditableStatus = {
   replaced: Translate.string('Replaced'),
   needs_submitter_confirmation: Translate.string('Needs submitter confirmation'),
   rejected: Translate.string('Rejected'),
+  accepted_submitter: Translate.string('Accepted by submitter'),
```

Comment:
```suggestion
  accepted_submitter: Translate.string('Accepted by submitter', 'Editable'),
```
Sorry for being an i18n-Nazi but I'd also add context to the remaining messages (or at least those that use past participles, those are tricky to translate w/o context in Slavic languages)